src/resources/postcss/_mixins.pcss

```diff
@@ -0,0 +1,15 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Utilities
+ *
+ * This file is just a clearing-house.
+ * Make partials (start with an underscore) for any actual css code.
+ */
+
+/* @todo: remove this and edit the files that depend on it when we create a mxins/_all.pcss in common-styles. */
```

Comment:
todo?

Reply:
Yeah - this is a reminder we should probably create an import in common-styles for this.